packages/connectors/connector-dingtalk-web/src/constant.ts

```diff
@@ -0,0 +1,56 @@
+import type { ConnectorMetadata } from '@logto/connector-kit';
+import { ConnectorConfigFormItemType, ConnectorPlatform } from '@logto/connector-kit';
+
+export const authorizationEndpoint = 'https://login.dingtalk.com/oauth2/auth';
+export const accessTokenEndpoint = 'https://api.dingtalk.com/v1.0/oauth2/userAccessToken';
+export const userInfoEndpoint = 'https://api.dingtalk.com/v1.0/contact/users/me';
```

Comment:
Found a few endpoints that related to getting user info. Would you like to attach doc link here so that others can review/maintain this in a easier way?

Reply:
The link for user info endpoint seems not right? The code implementation endpoint ends with `/me` and the docs endpoint ends with `/${unionId}`. Would you like to update the link? If the implementation `/me` is intended, could you please add more reference?